tests/auto/api.json

```diff
@@ -1932,6 +1951,66 @@
                 "success": "Bool",
                 "transactionId": "Int"
             }
+        },
+        "Zigbee.AddNetwork": {
+            "description": "Create a new Zigbee network for the given serialPort, baud rate and backend type. Get those information from the available Zigbee adapters.The channel mask is optional and defaults to all channels [11, 26]. The quietest channel from the given channel mask will be picked during network creation. The channel mask is a uint32 flag and the the bit number represents the channel which should enabled for scanning. All channels would be the value 0x07fff800.",
+            "params": {
+                "backendType": "$ref:ZigbeeBackendType",
+                "baudRate": "Uint",
+                "o:channelMask": "Uint",
+                "serialPort": "String"
+            },
+            "returns": {
+                "o:networkUuid": "Uuid",
+                "zigbeeError": "$ref:ZigbeeError"
+            }
+        },
+        "Zigbee.FactoryResetNetwork": {
+            "description": "Factory reset the network with the given networkUuid. The network does not have to be Online for this procedure, and all associated nodes and things will be removed permanently. Make sure the user realy wants to do this because this can not be undone.",
```

Comment:
Online -> online

no , before and

realy -> really. Probably that last sentence isn't even needed in here... "permanently" already says it's definitive... Then it's really up to the app use case on how important, risky, whatever it is for that particular use case.